build/build-image/Dockerfile

```diff
@@ -21,3 +21,8 @@ RUN export version=1.0.8 && \
 
 # Get rid of "go: disabling cache ..." errors.
 RUN mkdir -p /go && chgrp -R root /go && chmod -R g+rwX /go
+
+RUN export GOLANGCI_LINT_VERSION=v1.17.1 &&\
+    curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s $(GOLANGCI_LINT_VERSION) && \
+    mv ./bin/golangci-lint /usr/local/golangci-lint && \
```

Comment:
don't you want this in /usr/local/bin so it's in one of the default bin search paths?

Reply:
yes, I wanted put it in `/usr/local/bin` but mistakenly out it in `/usr/local`